frontend/src/component/executiveDashboard/Charts.tsx

```diff
@@ -124,9 +125,7 @@ export const Charts: VFC<IChartsProps> = ({
                     <FlagStats
                         count={summary.total}
                         flagsPerUser={
-                            showAllProjects
-                                ? (summary.total / users.total).toFixed(2)
```

Comment:
Sorry, this has taken me a while to realize, but **this** is what caused us to store the string 'NaN', right? 

Couldn't we have gotten around all the other changes here by instead doing something like:

```ts
const averageFlagsPerUserCalculation = summary.total / users.total
const averageFlagsPerUser = averageFlagsPerUserCalculation.isNan() ? 'N/A' : averageFlagsPerUserCalculation.toFixed(2)
```

and then using that value as the prop instead?

That way, we don't need to update any other files, we don't need to touch the hook, and we don't need to touch any tests, etc.

Reply:
I think it makes more sense to move the calculation to the summary hook inline with the rest of the calculations.

Reply:
Sure. You know the system better than I do, so I'll trust you on that.